make_data_sets.py

```diff
@@ -13,6 +13,221 @@
+        unit="metric tons of CO2 equivalent"
+    ),
+
+    "Toxic Releases": dict(
+        echo_type="TRI",
+        base_table="POLL_RPT_COMBINED_EMISSIONS",
+        table_name="TOXIC_RELEASES_MVIEW",
+        idx_field="REGISTRY_ID",
+        date_field="REPORTING_YEAR",
+        date_format="%Y"
+    ),
+
+    "CWA Violations": dict(
+        echo_type="NPDES",
+        base_table="NPDES_QNCR_HISTORY",
+        table_name="WATER_QUARTERLY_VIOLATIONS_MVIEW", 
+        idx_field="NPDES_ID",
+        date_field="YEARQTR",
+        date_format="%Y", 
+        agg_type="sum",
+        agg_col="NUME90Q", 
+        unit="effluent violations"
+    ),
+
+    "CWA Inspections": dict(
+        echo_type="NPDES",
+        base_table="NPDES_INSPECTIONS",
+        table_name="CLEAN_WATER_INSPECTIONS_MVIEW", 
+        idx_field="NPDES_ID",
+        date_field="ACTUAL_END_DATE", 
+        date_format="%m/%d/%Y",
+        agg_type="count", 
+        agg_col="STATE_EPA_FLAG",
+        unit="inspections"
+        # For possible later use in assessing state v federal
+    ),
+
+    "CWA Penalties": dict(
+        echo_type="NPDES",
+        base_table="NPDES_FORMAL_ENFORCEMENT_ACTIONS",
+        table_name="CLEAN_WATER_ENFORCEMENT_ACTIONS_MVIEW", 
+        idx_field="NPDES_ID",
+        date_field="SETTLEMENT_ENTERED_DATE", 
+        date_format="%m/%d/%Y",
+        agg_type="sum", 
+        agg_col="FED_PENALTY_ASSESSED_AMT",
+        unit="dollars"
+    ),
+
+    "SDWA Site Visits": dict(
+        echo_type="SDWA",
+        base_table="SDWA_SITE_VISITS",
+        table_name="SDWA_SITE_VISITS_MVIEW",
+        idx_field="PWSID",
+        date_field="SITE_VISIT_DATE",
+        date_format="%m/%d/%Y"
+    ),
+
+    "SDWA Enforcements": dict(
+        echo_type="SDWA",
+        base_table="SDWA_ENFORCEMENTS",
+        table_name="SDWA_ENFORCEMENTS_MVIEW",
+        idx_field="PWSID",
+        date_field="ENFORCEMENT_DATE",
+        date_format="%m/%d/%Y"
+    ),
+
+    "SDWA Public Water Systems": dict(
+        echo_type="SDWA",
+        base_table="SDWA_PUB_WATER_SYSTEMS",
+        table_name="SDWA_PUB_WATER_SYSTEMS_MVIEW",
+        idx_field="PWSID",
+        date_field="FISCAL_YEAR",
+        date_format="%Y"
+    ),
+
+    "SDWA Violations": dict(
+        echo_type="SDWA",
+        base_table="SDWA_VIOLATIONS",
+        table_name="SDWA_VIOLATIONS_MVIEW",
+        idx_field="PWSID",
+        date_field="FISCAL_YEAR",
+        date_format="%Y"
+    ),
+
+    "SDWA Serious Violators": dict(
+        echo_type="SDWA",
+        base_table="SDWA_SERIOUS_VIOLATORS",
+        table_name="SDWA_SERIOUS_VIOLATORS_MVIEW",
+        idx_field="PWSID",
+        date_field="FISCAL_YEAR",
+        date_format="%Y"
+    ),
+
+    # "SDWA Return to Compliance": dict(
+    #     echo_type="SDWA",
+    #     table_name="SDWA_RETURN_TO_COMPLIANCE",
+    #     idx_field="PWSID",
+    #     date_field="FISCAL_YEAR",
+    #     date_format="%Y"
+    # )
```

Comment:
This preset was there before but commented out. Normally I’d just drop stuff like that, but opted to keep it here (but reformatted) in case this was temporarily commented out. LMK if it should just be removed.

Reply:
Let's keep it, if only as a note to ourselves that this table exists in the database and that this is its configuration.